### DISCUSSION TITLE
I would like to ask how to create my own dataset to test Posec3d

### DISCUSSION THREAD

**Discussion by Sherryxingxing** [open-mmlab/mmaction2#2505] [Q&A]
I would like to ask how to create my own dataset to test Posec3d？

**Comment by cir7** [comment]
Hi, there are some tutorials about customizing PoseDataset:
1. [Customize keypoint format for PoseDataset](https://mmaction2.readthedocs.io/en/latest/advanced_guides/customize_dataset.html#customize-keypoint-format-for-posedataset)
2. [Custom Dataset Training with PoseC3D](https://github.com/open-mmlab/mmaction2/blob/main/configs/skeleton/posec3d/custom_dataset_training.md)
